### ISSUE TITLE
rootstat.py stopped working

### ISSUE THREAD

**Issue by ikatza** [LArSoft/larbatch#7]
Seems like I'll be the first to open an issue in here.

```
$ rootstat.py ../../developHEAD/ellint_pi_debug/g4.root

Opening ../../developHEAD/ellint_pi_debug/g4.root
Traceback (most recent call last):
  File "/cvmfs/larsoft.opensciencegrid.org/products/larbatch/v01_52_00/bin/rootstat.py", line 369, in <module>
    sys.exit(main(sys.argv))
  File "/cvmfs/larsoft.opensciencegrid.org/products/larbatch/v01_52_00/bin/rootstat.py", line 307, in main
    analyze(root, level, gtrees, gbranches, all)
  File "/cvmfs/larsoft.opensciencegrid.org/products/larbatch/v01_52_00/bin/rootstat.py", line 122, in analyze
    for branch in sorted(branches):
TypeError: '<' not supported between instances of 'TBranchElement' and 'TBranchElement'

```

I suspect this issue is related to migrating to python3.

**Comment by gartung** [comment]
@hgreenlee

**Comment by ikatza** [comment]
Allow me to give some more information. 

I'm running sbndcode v08_43_00, this in turns calls larbatch v01_52_01, where this issue is still present. 

Googling around I found these relevant links: 
https://stackoverflow.com/questions/51946150/django-typeerror-not-supported-between-instances-model-objects
https://docs.python.org/3/whatsnew/3.0.html#ordering-comparisons

Python 3 stopped trying to figure out how to order things. This needs to be defined.

I don't know what is the expected behaviour of ordering `branches` of type `<class 'ROOT.TObjArray'>`

I've tried replacing `sorted(branches)` with `sorted(branches, key=lambda x: (x.__str__())` and it works. Essentially sorting alphabetically, which is as arbitrary as not sorting at all.

From what I've seen there are three solutions: 

-  define `__lt__()` operator, somewhere, maybe in ROOT itself. 
- pass a meaningful lambda function to the key parameter (like I showed above)
- remove the sorted() functions.

If you guys can tell me whit respect to what the function should sort I can maybe provide a PR.
